source/includes/job-board/_applications.md

```diff
@@ -49,19 +49,19 @@ Please note that when submitting an application through this method, Greenhouse
 
 ### HTTP Request
 
-`POST https://api.greenhouse.io/v1/applications/`
+`POST https://api.greenhouse.io/v1/boards/{board_token}/jobs/{id}`
 
 ### Form Parameters
 
 Parameter | Description
 --------- | -----------
+board_token | Job Board URL token
```

Comment:
`board_token` is a URL parameter, so let's create a new URL parameters section for this and `id` (job post id)